protocol/x/affiliates/keeper/msg_server_test.go

```diff
@@ -132,3 +132,66 @@ func TestMsgServer_UpdateAffiliateTiers(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgServer_RegisterAffiliateInvalidSigner(t *testing.T) {
```

Comment:
Nit: let's move this to `protocol/x/affiliates/e2e` (similar to `protocol/x/clob/e2e`), since this is an e2e test, not just the message server.